lang/ruby/test/tool.rb

```diff
@@ -27,7 +27,7 @@ def initialize(proto, msg, datum)
     @datum = datum
   end
 
-  def call(message, request)
+  def call(message, _)
```

Comment:
Since this is in test code, could the unused argument be removed?

Reply:
@tjwp I've just noticed that removing the second argument causes the following error when running `share/test/interop/bin/test_rpc_interop.sh`.

```
Exception in thread "main" org.apache.avro.AvroRuntimeException: wrong number of arguments (given 2, expected 1)
        at org.apache.avro.ipc.generic.GenericRequestor.readError(GenericRequestor.java:66)
        at org.apache.avro.ipc.Requestor$Response.getResponse(Requestor.java:566)
        at org.apache.avro.ipc.Requestor$TransceiverCallback.handleResult(Requestor.java:366)
        at org.apache.avro.ipc.Requestor$TransceiverCallback.handleResult(Requestor.java:330)
        at org.apache.avro.ipc.Transceiver.transceive(Transceiver.java:73)
        at org.apache.avro.ipc.Requestor$TransceiverCallback.handleResult(Requestor.java:354)
        at org.apache.avro.ipc.Requestor$TransceiverCallback.handleResult(Requestor.java:330)
        at org.apache.avro.ipc.Transceiver.transceive(Transceiver.java:73)
        at org.apache.avro.ipc.Requestor.request(Requestor.java:152)
        at org.apache.avro.ipc.Requestor.request(Requestor.java:101)
        at org.apache.avro.tool.RpcSendTool.run(RpcSendTool.java:91)
        at org.apache.avro.tool.Main.run(Main.java:66)
        at org.apache.avro.tool.Main.main(Main.java:55)
```

So I'm going to rename the second argument with a preceding underscore just as other arguments, rather than remove that.